src/manifold/src/impl.h

```diff
@@ -144,8 +132,10 @@ struct Manifold::Impl {
         halfedgeTangent_[i][j] = meshGL.halfedgeTangent[4 * i + j];
     }
 
+    Vec<TriRef> triRef;
     if (meshGL.runOriginalID.empty()) {
-      // FIXME: should we do this?
+      // FIXME: This affects Impl::InitializeOriginal, and removing this
+      // apparently make things fail. Not sure if this is expected.
```

Comment:
Yeah, I think I remember what's going on there actually. I think it's benign, but I should probably refactor it at some point to make the code clearer.